kolibri/plugins/media_player/assets/src/views/customButtons.js

```diff
@@ -23,8 +24,4 @@ export class ForwardButton extends videojsButton {
   }
 }
 
-export class MimicFullscreenToggle extends videojsFullscreenToggle {
-  handleClick() {
-    this.player().trigger('mimicFullscreenToggled');
-  }
-}
+ForwardButton.prototype.controlText_ = 'Forward';
```

Comment:
Do 'Replay' and 'Forward' need to be internationalized?

Reply:
They're internationalized within Video.js.